src/main/java/com/amazon/inspector/jenkins/amazoninspectorbuildstep/AmazonInspectorBuilder.java

```diff
@@ -92,11 +93,15 @@ public class AmazonInspectorBuilder extends Builder implements SimpleBuildStep {
     private Job<?, ?> job;
 
     @DataBoundConstructor
-    public AmazonInspectorBuilder(String archivePath, String archiveType, String sbomgenPath, boolean osArch, String iamRole,
+    public AmazonInspectorBuilder(String archivePath, String artifactPath, String archiveType, String sbomgenPath, boolean osArch, String iamRole,
                                   String awsRegion, String credentialId, String awsProfileName, String awsCredentialId,
                                   String sbomgenMethod, String sbomgenSource, boolean isThresholdEnabled, int countCritical,
                                   int countHigh, int countMedium, int countLow, String oidcCredentialId) {
-        this.archivePath = archivePath;
+        if (artifactPath != null && !artifactPath.isEmpty()) {
+            this.archivePath = artifactPath;
```

Comment:
Would it make sense to rename `this.archivePath` to `this.artifactPath` for future clarity?

Reply:
We can't, if we change it it would break users currently active pipelines. I think it would be a good idea to communicate that its deprecated and completely rename it in the future though.